models/src/main/thrift/events/fastly/event.thrift

```diff
@@ -9,7 +9,7 @@ enum EventType {
 }
 
 struct ContentDecachedEvent {
-    1: required string contentId
+    1: required string contentPath
```

Comment:
This rename allowed as Facebook is the only consumer and that's my job to update.